bootstrap/bootstrap.py

```diff
@@ -38,36 +38,42 @@ def ingest():
     with open("config.yaml", 'r') as ymlfile:
         cfg = yaml.safe_load(ymlfile)
 
-    # Server parameters
-    ims_host = cfg["Server"]["ims_host"]
-    ims_endpoint_jwt = cfg["Server"]["ims_endpoint_jwt"]
+    # Get the platform url
+    platform_gateway_url = cfg['Platform']['platform_gateway']
 
-    # Enterprise parameters used to construct JWT
     api_key = cfg["Enterprise"]["api_key"]
     org_id = cfg["Enterprise"]["org_id"]
-    client_secret = cfg["Enterprise"]["client_secret"]
-    tech_acct = cfg["Enterprise"][ "tech_acct"]
-    priv_key_filename = cfg["Enterprise"]["priv_key_filename"]
 
-    # read private key from file
-    priv_key_file = open(priv_key_filename, "r")
-    priv_key = priv_key_file.read()
-    priv_key_file.close()
+    ims_token = cfg['Platform']['ims_token']
 
-    # Get the platform url
-    platform_gateway_url = cfg['Platform']['platform_gateway']
+    if ims_token is None:
+        # Server parameters
+        ims_host = cfg["Server"]["ims_host"]
```

Comment:
I see two quote formats used for cfg, single and double quotes? (e.g. `cfg['Platform']` and `cfg["Server"]`)